### ISSUE TITLE
Checkbox lacks contrast ratio when unchecked

### ISSUE THREAD

**Issue by multiwebinc** [shadcn/ui#223]
The border color of the checkbox button has a low contrast ratio when unchecked. Usually it is not important that the border color of an element have a high contrast ratio since the content is still visible, but in the case of checkboxes, there is no inner content.

Dark:

![image](https://user-images.githubusercontent.com/901732/233426039-0d4e60f7-f1e9-483c-9825-9f30827baad0.png)

Contrast ratio: 1.36

![image](https://user-images.githubusercontent.com/901732/233425255-83faba8c-c5a5-411b-bf62-aafb041eb679.png)

Light:

![image](https://user-images.githubusercontent.com/901732/233425958-ed85ede0-c07c-4285-8cd3-d02b6263ac9c.png)

Contrast ratio: 1.23

![image](https://user-images.githubusercontent.com/901732/233425742-cc2bb90e-a9aa-418b-89aa-e81a3e85b966.png)
